src/zfs-9p/util.c

```diff
@@ -78,6 +78,7 @@ void *
 init_9p(void * v){
 		IxpConn *conn;
 
+        //char *address = "tcp!*!5640";
```

Comment:
For debugging. If you uncomment that line, you can use zfs-9p as ufs. Just an experiment. Remember that this code is an intermediate version, there are a lot of code only for debugging purpose.